agent/consul/fsm/fsm.go

```diff
@@ -197,7 +197,11 @@ func (c *FSM) Restore(old io.ReadCloser) error {
 				return err
 			}
 		default:
-			return fmt.Errorf("Unrecognized msg type %d", msg)
+			if msg > 63 {
```

Comment:
Optional comment: I find `msg >= 64` more readable than `msg > 63`.

Will ask engineering for review!

Reply:
Agreed with favoring >=64

Reply:
Thank you for the review. I have modified the check to make it readable as suggested.